ts/theme/variables.ts

```diff
@@ -142,11 +149,22 @@ const customVariables = Object.assign(materialVariables, {
   textBoldWeight: "700" as FontWeight,
   textLinkWeight: "600" as FontWeight,
 
+  // Item
```

Comment:
what is `Item`? again, _item_ is a very generic word, is like saying "thing"

Reply:
Also this file must contain only generic (read cross-app) values.